src/AtmosParameters.jl

```diff
@@ -12,18 +12,74 @@ AtmosSGS.c_1_KASM(ps::AbstractEarthParameterSet)        = AtmosSGS.c_a_KASM(ps)*
 AtmosSGS.c_2_KASM(ps::AbstractEarthParameterSet)        = AtmosSGS.c_e2_KASM(ps)+2*AtmosSGS.c_1_KASM(ps)
 AtmosSGS.c_3_KASM(ps::AbstractEarthParameterSet)        = AtmosSGS.c_a_KASM(ps)^(3/2)
 
-Microphysics = CLIMAParameters.Atmos.Microphysics
 # Microphysics parameters
-Microphysics.MP_n_0(::AbstractEarthParameterSet)          = 8e6 * 2
-Microphysics.C_drag(::AbstractEarthParameterSet)          = 0.55
-Microphysics.τ_cond_evap(::AbstractEarthParameterSet)     = 10
-Microphysics.τ_sub_resub(::AbstractEarthParameterSet)     = 10
-Microphysics.q_liq_threshold(::AbstractEarthParameterSet) = 5e-4
-Microphysics.τ_acnv(::AbstractEarthParameterSet)          = 1e3
-Microphysics.E_col(::AbstractEarthParameterSet)           = 0.8
-Microphysics.a_vent(::AbstractEarthParameterSet)          = 1.5
-Microphysics.b_vent(::AbstractEarthParameterSet)          = 0.53
-Microphysics.K_therm(::AbstractEarthParameterSet)         = 2.4e-2
-Microphysics.D_vapor(::AbstractEarthParameterSet)         = 2.26e-5
-Microphysics.ν_air(::AbstractEarthParameterSet)           = 1.6e-5
-Microphysics.N_Sc(ps::AbstractEarthParameterSet)          = Microphysics.ν_air(ps)/Microphysics.D_vapor(ps)
+Microphysics = CLIMAParameters.Atmos.Microphysics
+
+# general
+Microphysics.C_drag(::AbstractEarthParameterSet)  = 0.55
```

Comment:
Will this result in type problems further down the stack? (I worry when I see raw decimals)

Reply:
We've decided to wrap everything returned from these functions with `FT`, so that we can make a single global `param_set = EarthParameterSet()` per driver. Otherwise, we'd need to create `param_set = EarthParameterSet{FT}()` in multiple functions/locations. This was a trade-off I discussed with Simon a while back, and we both liked the idea of having a single struct at the top of each driver.

Reply:
Also, the float-type is really orthogonal to the values returned from a parameter set, and it seemed like including `FT` would entangle things that really didn't have to be.

Reply:
Thanks for the info. Figured you thought of this.